src/components/input/Input.style.tsx

```diff
@@ -8,7 +8,7 @@ export const StyledInputContainer = styled.div`
 `
 
 export const StyledLabel = styled.label<IInput>`
-  color: ${({ errorType }) => (errorType ? '#ff0000' : '#5849ff')};
+  color: ${({ errorType }) => (errorType ? '#ff0000' : 'var(--color-main)')};
```

Comment:
에러 색상 추가되었는데, 계속 쓰일 것 같으니까 글로벌 스타일에도 추가해주시고, 해당 코드에도 반영해주시면 좋을 것 같아요

```suggestion
--color-caution : #F00